homework/transform-containers/transformContainers.cpp

```diff
@@ -0,0 +1,15 @@
+#include "transformContainers.hpp"
+#include <algorithm>
+#include <iterator>
+
+std::map<int, std::string> removeDuplicateAndTranformToMap(std::list<std::string> list, std::deque<int> deque) {
+    list.sort();
+    std::sort(deque.begin(), deque.end());
+    list.erase(std::unique(list.begin(), list.end()), list.end());
+    deque.erase(std::unique(deque.begin(), deque.end()), deque.end());
+    std::map<int, std::string> resultMap;
+    std::transform(deque.begin(), deque.end(), list.begin(), std::inserter(resultMap, resultMap.end()), [](int first, std::string second) {
+        return std::make_pair(first, second);

```

Comment:
You should consider, what can happen if these two containers will have different size

Reply:
Size of list must be equal or bigger then deuqe, then algorithm will work. Second option is to reject all input which has different sizei